consensus/scheme/rolldpos/rolldposctx.go

```diff
@@ -486,6 +486,7 @@ func (ctx *rollDPoSCtx) Commit(msg interface{}) (bool, error) {
 	case nil:
 		break
 	default:
+		log.L().Error("error when committing the block", zap.Error(err))
```

Comment:
if the returned error is logged by upstream caller, then no need to add this
pls make sure that's not the case